asyncscope.md

```diff
@@ -105,7 +105,7 @@ int main() {
 }
 ```
 
-The newly introduced `async_scope` object allows us to control the lifetime of the dynamic work we are spawning.
+The newly introduced `async_scope` object allows us to scope the lifetime of the dynamic work we are spawning.
 We can wait for all the work that we spawn to be complete before we destruct the objects used by the parallel work.
 
 Please see below for more examples.
```

Comment:
It might be useful to mention as an example async code that runs senders and continuations in GUI event loops. The significant thing in those is that the event loop is created non-locally, and not destroyed by the async code. It's usually created and destroyed in main(), which is certainly not the place where you write your async code. Thus it's not-at-all straightforward to use ensure_started and start_detached and sync_wait and fiber_wait etc. in that code, and it really really needs an async_scope to tie the lifetime of the dynamic work into the right scope, which is basically the orchestrator that sets up the work, and the lifetime of that orchestrator is nested under the event loop's lifetime, but other than that the orchestrator doesn't really care about the lifetime of the event loop. Using async_scope makes it very easy to scope the lifetimes of the async work properly.